scripts/zkai_d128_two_slice_accumulator_backend_gate.py

```
@@ -0,0 +1,913 @@
+    expect_equal(handle.get("schema"), VERIFIER_HANDLE_SCHEMA, "verifier handle schema")
+    expect_equal(handle.get("claim_boundary"), CLAIM_BOUNDARY, "verifier handle claim boundary")
+    expect_equal(handle.get("accepted"), True, "verifier handle accepted")
+    expected = verifier_handle(artifact)
+    expect_equal(handle, expected, "verifier handle")
+
+
+def _validate_recursive_status(payload: dict[str, Any]) -> None:
+    status = require_object(payload.get("recursive_or_pcd_status"), "recursive or PCD status")
+    expect_equal(status.get("result"), RECURSIVE_OR_PCD_RESULT, "recursive or PCD result")
+    if status.get("recursive_outer_proof_claimed") is not False:
+        raise D128TwoSliceAccumulatorBackendError("recursive outer proof claimed without backend")
```

Ruff (0.15.12)

[warning] Avoid specifying long messages outside the exception class

(TRY003)